packages/stores/src/providers/WebsiteStatusProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+import { observer } from 'mobx-react-lite';
+import useStore from '../useStore';
+import merge from 'lodash.merge';
+
+const WebsiteStatusProvider = observer(({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, subscribe, unsubscribe } = useSubscription('website_status');
+    const {
+        client: { is_authorize },
+        website_status: { update },
+    } = useStore();
+
+    useEffect(() => {
+        if (data) {
+            const { website_status } = data;
+            if (website_status) update(prev => merge(prev, website_status));
+        }
+    }, [update, data]);
+
+    // handles case where the response from website_status is different when client is logged in/out, we need to re-subscribe if client is logging in
```

Comment:
@adrienne-deriv Let's remove the comments in the code, The comments will get outdated over time and will mislead the next dev. 

You can put comments on the PR like you [already](https://github.com/binary-com/deriv-app/pull/8702/files#r1206272108) do so devs can find it through git blame on the PR at that time.